src/components/views/elements/AccessibleButton.tsx

```diff
@@ -63,8 +63,7 @@ type DynamicElementProps<T extends keyof JSX.IntrinsicElements> = Partial<
  * onClick:  (required) Event handler for button activation. Should be
  *           implemented exactly like a normal onClick handler.
  */
-type IProps<T extends keyof JSX.IntrinsicElements> = DynamicHtmlElementProps<T> & {
-    inputRef?: React.Ref<Element>;
+export type Props<T extends keyof JSX.IntrinsicElements> = DynamicHtmlElementProps<T> & {
```

Comment:
> I think the first rule only makes sense if you're going to honour the second.

The code style doesn't say that though - one is a MUST the other a SHOULD

Reply:
well frankly I call BS on that. Exporting a calls called `Props` is just stupid. If you think that means the code style needs changing, then let's do it, rather than slavishly doing something that makes literally no sense.